.github/workflows/e2e-nvidia-a10g-x4.yml

```diff
@@ -123,17 +123,17 @@ jobs:
           python3.11 -m pip cache remove llama_cpp_python
           CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3.11 -m pip install --force-reinstall --no-binary llama_cpp_python -c constraints.txt llama_cpp_python
           python3.11 -m pip install bitsandbytes
-          
           python3.11 -m pip install instructlab-training[cuda]
-
+          python3.11 -m pip install -r requirements-vllm-cuda.txt
           python3.11 -m pip install .
 
       - name: Run e2e test
         env:
           HF_TOKEN: ${{ secrets.HF_TOKEN }}
         run: |
           . venv/bin/activate
-          ./scripts/basic-workflow-tests.sh -mFMT
+
+          ./scripts/basic-workflow-tests.sh -evmFMT
```

Comment:
This won't work. The mixtral we're using is gguf. You'll have to update the mixtral handling too